locales/en/common.json

```diff
@@ -25,5 +26,14 @@
   "type_value_to_confirm_plain": "Type {{value}} to confirm",
   "unexpected_error": "There was a problem processing the request. Please try again.",
   "updated": "Updated",
-  "we_cannot_find_the_page_you_are_looking_for": "We can’t find the page you’re looking for."
+  "we_cannot_find_the_page_you_are_looking_for": "We can’t find the page you’re looking for.",
+  "bootstrap_server": "Bootstrap server",
+  "page": "Page",
```

Comment:
Where this text displays in the UI, it shouldn't have a capital "P"
![image](https://user-images.githubusercontent.com/21063328/157551711-fa5cbb7f-1d69-4d7d-be51-eb84b8ff508d.png)